packages/vc-verification/src/models/credential-resolver.ts

```diff
@@ -0,0 +1,6 @@
+/**
+ * Storage settings to fetch issuer's credential
+ */
+export interface CredentialResolverSettings {
+  ipfsApi: string;
```

Comment:
Minor point but I think `Url` is a bit more accurate and in line with what we use elsewhere
```suggestion
  ipfsUrl: string;
```